python/app/syntax_highlighter.py

```diff
@@ -8,26 +8,17 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
+import builtins
```

Comment:
this import is necessary for the line [54](https://github.com/shotgunsoftware/tk-multi-pythonconsole/pull/38/files#diff-6f5e365ad2d119b73302d5530ced3ede5d4abc402e2d4c5cebf7e11a870a95e7L54)

Reply:
Can't find it?!

Reply:
Actually, it is used for syntax highlighting in Python:
```
class PythonSyntaxHighlighter(QtGui.QSyntaxHighlighter):
    """Syntax highlighter for the Python language."""
    ...
    # Python builtins
    builtins = dir(builtins)
    ...
    rules += [
         (r"\b%s\b" % w, 0, self._style("builtin"))
         for w in PythonSyntaxHighlighter.builtins
    ]
    ...
```
So, the builtins module is a complete Python object, not a list. If we tried to iterate over it directly in the highlighting rules, it wouldn't work, because it's not iterable in the way we expect.
Converting it to a list using dir() simplifies the operation, as you can easily iterate over it and create regular expressions with each name in the list.